src/components/ScrollToTop/ScrollToTop.js

```diff
@@ -20,7 +20,6 @@ export const ScrollToTop = (props) => {
     window.scroll({
       left: 0,
       top: window.pageYOffset - scrollStep,
-      behavior: 'smooth',
     });
   };
```

Comment:
- Based on what I understand, this method will scroll the same pixel, disregard of the height of the page, which means if our page is big, it will still takes a long time to scroll back to top 
- how about use ```scrollIntoView``` which might takes const time to scroll back to top
```
  const handleClick = event => {
    const anchor = (event.target.ownerDocument || document).querySelector('#back-to-top-anchor');

    if (anchor) {
      anchor.scrollIntoView({ behavior: 'smooth', block: 'center' });
    }
  };```

Reply:
Done